src/ui/components/Explorer.js

```diff
@@ -196,7 +196,7 @@ export class Explorer extends React.Component<ExplorerProps, ExplorerState> {
     const sortedNodes = sortBy(nodes, (n) => -n.credSummary.cred);
     const total = sum(nodes.map((n) => n.credSummary.cred));
     return (
-      <div style={{width: "1000px", margin: "0 auto"}}>
+      <div style={{width: "1000px", margin: "0 auto", background: "white"}}>
```

Comment:
Can we have a responsive width? What happens if you set it to 100%?

Reply:
I'd argue that this is out of scope for this PR. This line change essentially ensures visual parity with the existing Explorer. Discussion about styling for the explorer is ongoing and this will be addressed within those changes.

Reply:
80% improves things but there are still some issues: https://github.com/sourcecred/sourcecred/pull/1989/commits/44932129468593dea9704a1e3a8aeb4a2d1671b0